generators/ruby/faraday/templates/README.mustache

```diff
@@ -9,32 +9,37 @@ Documentation can be found at https://documentation.onfido.com
 
 This version uses Onfido API v3.6 and is compatible with Ruby 2.4 onwards. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
 
-## Installation
+## Installation & Usage
 
-Add this line to your application's Gemfile:
+### Requirements
+
+Ruby {{{generatorLanguageVersion}}}
+
+### Installation
 
 ```ruby
-gem 'onfido', '~> 2.0.1'
+gem {{{gemName}}}, '~> 2.0.1'
 ```
 
-## Getting started
-
 Configure with your API token and region:
 
 ```ruby
-onfido = Onfido::API.new(
-  api_key: ENV['ONFIDO_API_KEY'],
-  # Supports :eu, :us and :ca. Previously defaulted to :eu.
-  region: :eu
-)
+require {{{gemName}}}
+
+Onfido.configure do |config|
+  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.region = config.region[:EU]
```

Comment:
@DavidMealha is there any better way to define an enum in ruby?

Reply:
Where is this coming from in the first place? Is it auto-generated?

Reply:
Nevermind, I found it below.

The way it was implemented is the most common, but you could also add the values as constants:
```
module {{moduleName}}
  class Configuration
    EU_REGION = 'eu'
    US_REGION = 'us'
    CA_REGION = 'ca'
  end
end
```

and then use it in this line like this:
```
config.region = config::EU_REGION
```

Reply:
Ok, not sure if i'd prefer accessing it like it's implemented now. What do you think?

Reply:
No strong preference either 😁